src/main/kotlin/com/meltwater/docker/compose/DockerCompose.kt

```diff
@@ -55,21 +57,44 @@ class DockerCompose private constructor(
 
         private val ENVIRONMENT_VARIABLE_SEPARATOR = Pattern.compile("=")
 
-        private fun initializeNewDockerComposeSession(classPathYmlResource: String, env: HashMap<String, String>): String {
+        private val dockerCompose: String = findDockerCompose("docker-compose")?.toString() ?: "docker compose"
+
+        private fun findDockerCompose(name: String): Path? {
+            val pathSeparator = System.getProperty("path.separator")
+            val rawPath = System.getenv("PATH")
+            LOGGER.debug("Looking for executable in path. {}={}, {}={}", "executable", name, "path", rawPath)
+            val path = rawPath.split(pathSeparator.toRegex()).dropLastWhile { it.isEmpty() }.toTypedArray()
+            return path
+                .map { pathEntry: String ->
+                    Paths.get(
+                        pathEntry,
+                        name
+                    )
+                }.firstOrNull {
+                    Files.exists(it)
+                }
+        }
```

Comment:
We parse the `${PATH}` environment variable, and check each directory in path in-order, to see if it contains a file named `docker-compose`. It might be a good idea to check for `docker-compose.exe` on Windows, but I haven't added that discrimination yet. Though we use the correct path separator per platform (`:` for Unix/macOS, `;` for Windows)